akirachix/Upile-Informational
# Updates about section

What is this?
This PR updates the About section to improve its alignment, responsiveness, and styling in line with the latest design requirements.
Why this?
To enhance the user interface by ensuring the About section is visually consistent across various screen resolutions and devices.
How are you doing this?
List of updates for features, enhancements, or bugfixes:
Adjusted the layout, alignment, and spacing of the About section to maintain uniformity and responsiveness.
Improved text alignment and container spacing to optimize readability and design flow.
Type of change
Specify the kind of PR/MR. Please delete options that are not relevant:
Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)
Breaking change (fix or feature that would cause existing functionality to not work as expected)
This change requires a documentation update
Optimization
Improvement
Bug fix
[ ] Improvement
How can it be tested?
About Section:
Verify that the text, alignment, and spacing are consistent across different screen sizes.
Test the responsiveness to ensure that the layout adapts well on various devices.
Confirm that the content remains visually aligned and well-structured, regardless of the device or screen size.

**Screenshots**
![Screenshot from 2024-09-17 20-14-00](https://github.com/user-attachments/assets/1b6b7c10-495b-4647-aff0-73a93ed7b131)


# Where is it documented?
- List JIRA tickets
https://wambuinatalygakenye.atlassian.net/browse/UP-117?atlOrigin=eyJpIjoiOGUzMDA2ZGYxMzllNDc3MTk4ZjVkMDMwMDJkZmY0ZTAiLCJwIjoiaiJ9
